src/stories/Sandbox.stories.tsx

```diff
@@ -154,6 +154,10 @@ Sandbox.args = {
       isRequired: true,
       defaultValue: 'compare',
       divideAfter: true,
+      copyToClipboard: true,
+      validateOnChange: async (value) => {
```

Comment:
Do we want to be able to define the debounce time here? If so, could do something like

`
asyncValidator:{
timeout: number,
handler: async () => {}
} 
`

Reply:
Sounds good! I'll just move anything to do with the validation into an `inputValidation` prop